migrations/Version20201112220914.php

```diff
@@ -36,6 +26,6 @@ public function up(Schema $schema): void
 
     public function down(Schema $schema): void
     {
-        // this down() migration is auto-generated, please modify it to your needs
+        $this->throwIrreversibleMigrationException();
```

Comment:
What about dropping the table instead?

Reply:
I don't like down() migrations and I don't think we want to do this anyway